### ISSUE TITLE
Implementing a tag cloud

### ISSUE THREAD

**Issue by rupalkhilari** [electology/approval_frame#149]
@balajisubr suggested implementing a tag cloud for the app. It could perhaps give the audience some insight on the most popular categories of questions are in the app. Details are in the forum discussion here: https://groups.google.com/forum/#!topic/ces-software/UU7YCfpNzUg . 

@balajisubr do you propose that the tags in the tag cloud are clickable, filtering the polls based on the tag ?

**Comment by rupalkhilari** [comment]
As discussed in the forum, this feature would start out with having the following functions:

1. Letting the user create and assign tags to polls.
2. Displaying the tag cloud, and linking to it from the home page.
3. Displaying the filtered poll results on clicking the tag.

**Comment by balajisubr** [comment]
@rupalkhilari How do you want to show the existing tags for a poll in the edit page? Should they be tokenized and displayed in the text box? 

Also, what is the placement of the tag cloud for both desktop and mobile view?

**Comment by rupalkhilari** [comment]
@balajisubr yes, tokenized words in a text field sounds like a good idea. Also I think we could link to the tag cloud from the home page on the top should work for now.
